terraform/versions.tf

```diff
@@ -1,4 +1,10 @@
-
 terraform {
   required_version = ">= 0.12"
```

Comment:
Now that `0.13` is out and we haven't done testing, does this requirement still make sense?

Reply:
You are correct!  See commit 4bb72cf.